### PR TITLE
Fixes describe-bindings for keys that are bound to keymaps.

### PR DESCRIPTION
**Pull request by mychris**
With the latest change that the frame-multiplexer prefix key `C-z` is now a `keymap` (#1237), `describe-bindings` is broken, because it can't handle this case.

The implementation is now recursive and traverses each keymap which is bound to a key. In the overview, those "sub-keymaps" are shown under the section of the actual keymap and the prefix is added to each command. For `C-z` it looks like this:

```
Global (*global-keymap*)
key             binding
---             -------
C-n             next-line
...
C-z c           frame-multiplexer-create-with-new-buffer-list
C-z d           frame-multiplexer-delete
C-z p           frame-multiplexer-prev
C-z n           frame-multiplexer-next
C-z r           frame-mulitplexer-rename
```

### CODE REVIEW

File: src/commands/help.lisp [hunk 1]
```diff
@@ -24,19 +24,26 @@
...
-                               (lambda (kseq command)
-                                 (unless (equal "UNDEFINED-KEY" (symbol-name command))
-                                   (format s "~va~(~a~)~%"
-                                           column-width
-                                           (keyseq-to-string kseq)
-                                           (symbol-name command)))))
-              (setf keymap (keymap-parent keymap))
-              (terpri s))))
+    (labels ((create-describe-func (key-prefix)
+               #'(lambda (kseq command)
```
Comment (vindarel) [inline src/commands/help.lisp:28]: style: this `#'` before a lambda is from old times and not necessary.

Comment (mychris) [reply src/commands/help.lisp]: Thanks, fixed.

**Comment by mychris** [comment]
Maybe it's also worth thinking about `traverse-keymap`. Otherwise every single user of it needs to handle the keymap case. The prefix command case is handled there.
So maybe this PR should be dropped and traverse-keymap is the right place to fix?

**Comment by mychris** [comment]
Yes, the fix for `traverse-keymap` is much simpler and I think the right thing to do. I pushed another commit.

**Comment by cxxxr** [comment]
@mychris Can I merge this PR?

**Comment by mychris** [comment]
Yes, I think this fix is good now.

**Pull request merged**